### PR TITLE
New developer setup

### LINKED ISSUE #387: update "running the platform" doc
Plenty of old information in https://renku.readthedocs.io/en/latest/developer/setup.html -- clean it up and give instructions for deploying with helm.

### PR DESCRIPTION
**Pull request by leafty**
Resolves #387 

Removed the old setup page and took some bits from it to put in the minikube doc.

### CODE REVIEW

**Comment by rokroskar** [comment]
I think the "Running the platform" section should have just a few lines of helm commands to get renku running on a k8s cluster. There should then be a separate developer doc for running the platform from checked out source. It could even be on the same page, but I just mean something like:

"Create a `local-values.yaml` file specifying the values for x,y,z. Then run `helm upgrade --install renku --version 123 -f local-values.yaml`"

**Comment by leafty** [comment]
We can think of having harmless sensible values to deploy from a helm chart release with just a one liner, but the main point of a developer page is to set things up for development.

**Comment by rokroskar** [comment]
True -- so should we have an admin section?

File: docs/developer/setup.rst [hunk 1]
```diff
@@ -57,124 +36,109 @@ Get the platform
...
-Renku is deployed using Docker containers. For production deployments we
-include kubernetes helm charts; development environments can be created
-using `docker-compose`.
+Use ``pipenv``:
 
+.. code-block:: console
 
-Kubernetes
-^^^^^^^^^^
+    $ pipenv install
```
[the remark this answers is not in the document]
Comment (leafty) [reply docs/developer/setup.rst]: `make minikube-depkoy` does rely on `pipenv` and `chartpress`

[the remark this answers is not in the document]
Comment (rokroskar) [reply docs/developer/setup.rst]: aha ok, I wasn't sure

**Pull request merged**